modules/ocf_puppet/files/write-otp-config

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Inform me, how is this script intended to be run? Is this supposed to be run by the user or some other job?